packages/api/src/codegen/languages/typescript/index.ts

```diff
@@ -79,6 +80,26 @@ function handleExecFailure(err: Error, opts: InstallerOptions = {}) {
   throw err;
 }
 
+async function detectPackageManager(installDir: string): Promise<string> {
```

Comment:
Thoughts on using [preferred-pm](https://npm.im/preferred-pm) for this? They seem to have resolved the issue I opened last year that prevented us from implementing this.

https://github.com/zkochan/packages/issues/180

Reply:
done... unfortunately it doesn't support deno :(